TPCC bench failed without partitioning on 30 nodes

**Describe the problem**

TPCC bench failed without partitioning on 30 nodes 

**To Reproduce**

ran `bin/roachtest bench '^tpccbench/nodes=30/cpu=16$$' --wipe=false --user=andy`

We set ramp duration to 8hs:

`+++ b/pkg/cmd/roachtest/tpcc.go
@@ -414,7 +414,7 @@ func loadTPCCBench(
        // generation doesn't actually run.
        cmd = fmt.Sprintf("ulimit -n 32768; "+
                "./workload run tpcc --warehouses=%d --split --scatter %s "+
-               "--duration=3m --tolerate-errors {pgurl:1}", b.LoadWarehouses, partArgs)
+               "--duration=8h --tolerate-errors {pgurl:1}", b.LoadWarehouses, partArgs)
        if out, err := c.RunWithBuffer(ctx, c.l, loadNode, cmd); err != nil {
                return errors.Wrapf(err, "failed with output %q", string(out))`

Observed:
The restore took 48 minutes.


`: exit status 1
: exit status 1skipping cluster wipe
--- FAIL: tpccbench/nodes=30/cpu=16 (32099.78s)
	test.go:570,cluster.go:1318,tpcc.go:663,tpcc.go:333: pq: setting updated but timed out waiting to read new value
FAIL`

**Expected behavior**
I expected tpcc to pass. 

**Additional data / screenshots**
Roachprod destroyed the cluster before I could grab the logs 

**Environment:**
 - CockroachDB version 2.1.beta 10/08

--- a/pkg/sql/set_cluster_setting.go
+++ b/pkg/sql/set_cluster_setting.go
@@ -171,7 +171,7 @@ func (n *setClusterSettingNode) startExec(params runParams) error {
 	}
 	errNotReady := errors.New("setting updated but timed out waiting to read new value")
 	var observed string
-	err := retry.ForDuration(1*time.Second, func() error {
+	err := retry.ForDuration(10*time.Second, func() error {
 		observed = n.setting.Encoded(&execCfg.Settings.SV)
 		if observed != expectedEncodedValue {
 			return errNotReady

